react/features/app/getRouteToRender.web.js

```diff
@@ -75,7 +76,11 @@ function _getWebWelcomePageRoute(state) {
 
     if (isWelcomePageEnabled(state)) {
         if (isSupportedBrowser()) {
-            route.component = WelcomePage;
+            if (interfaceConfig.CUSTOM_WELCOME_PAGE_URL) {
```

Comment:
Interface config is deprecated. Please move this to config.js. In fact, since this is a new welcome page related setting we can start a section:

```
welcomePage: {
    enabled: true,
    customUrl: 'https://...'
```